### DISCUSSION TITLE
Change body direction from RTL to LTR

### DISCUSSION THREAD

**Discussion by luizsilva-dev** [4xmen/xshop#18] [Q&A] [answered]
How can I change the body direction from RTL to LTR?

**Comment by A1Gard** [comment, marked as answer]
Yes, of course, you can customize `scss` as you wish. 

admin panel:

https://github.com/4xmen/xshop/blob/main/resources/sass/app.scss

and website:

https://github.com/4xmen/xshop/blob/main/resources/theme/theme.scss

don't forget to run laravel-mix after changes
